scripts/update-dependencies/Program.cs

```diff
@@ -1,68 +1,76 @@
-// Copyright (c) .NET Foundation and contributors. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using Microsoft.DotNet.VersionTools;
-using Microsoft.DotNet.VersionTools.Automation;
-using Microsoft.DotNet.VersionTools.Dependencies;
 using System;
-using System.Collections.Generic;
 using System.Diagnostics;
-using System.IO;
 using System.Linq;
-using System.Text.RegularExpressions;
+using System.Net.Http;
 using System.Threading.Tasks;
+using System.Xml;
+using Microsoft.DotNet.VersionTools.Automation;
+using Microsoft.Extensions.Configuration;
 
 namespace Microsoft.Dotnet.Scripts
 {
     public static class Program
     {
         private static readonly Config config = Config.Instance;
 
-        public static void Main(string[] args)
+        public static async Task Main(string[] args)
         {
             Trace.Listeners.Add(new TextWriterTraceListener(Console.Out));
 
-            if (!ParseArgs(args))
-                return;
+            ParseArgs(args);
 
-            CreatePullRequest();
+            await CreatePullRequest();
+        }
+
+        private static void ParseArgs(string[] args)
+        {
+            var builder = new ConfigurationBuilder().AddCommandLine(args);
+            var configRoot = builder.Build();
+            Config.Instance.Load(configRoot);
```

Comment:
Remove `Config.Instance` and just have a local `Config` field. Also the only advantage of using `Configuration` here over `CommandLineUtils` is so you can bind it. Consider using that. I'd still prefer using CommandLineUtils here, just seems less unusual as a way to parse command line args